css/main.css

```diff
@@ -0,0 +1,620 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+}
+
+body {
+  background-color: #f2f2f2;
+  overflow-x: hidden;
+}
+
+.top_navigation {
+  height: 2rem;
+  background: #3e3e3e;
+}
+
+.top_navigation ul {
+  display: flex;
+}
+
+.top_navigation ul li {
+  color: white;
+  margin-right: 5%;
+  list-style-type: none;
+}
+
+.lower_navigation {
+  height: auto;
+  display: flex;
+  align-items: center;
+  background: #fff;
+}
+
+.lower_navigation ul {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-right: 8%;
+}
+
+.lower_navigation ul li {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.footer_paragraph_gmail_home a {
+  color: #000;
+}
+
+.organisation a {
+  font-weight: bold;
+  text-decoration: none;
+  color: #000;
+}
+
+.footer_paragraph_gmail a {
+  color: #fff;
+}
+
+.email_footer a {
+  text-decoration: none;
+  color: white;
+}
+
+.down_part span a {
+  color: white;
+  text-decoration: underline;
+  font-size: 18px;
+  font-family: 'lato', sans-serif;
+}
+
+.lower_navigation ul li a {
+  text-decoration: none;
+  color: #3e3e3e;
+  font-family: 'poppins', sans-serif;
+  font-weight: 500;
+  font-size: 17px;
+}
+
+.lower_navigation ul li:nth-child(6) {
+  border: 4px solid red;
```

Comment:
- [ Required ] Please consider using the exact colors used in the `Behance` design in your design. The color used on the border of the button on the menu is `#EC5242` not `red` kindly make sure that these are the colors you use on you design.

<img src="https://user-images.githubusercontent.com/16392046/212085940-fc8b4c90-2742-4827-babe-07f9595332d9.png" width="375px" />